include/SDL_mouse.h

```diff
@@ -92,6 +92,41 @@ typedef enum
     SDL_MOUSEWHEEL_SOURCE_OTHER_KINETIC,
 } SDL_MouseWheelSource;
 
+typedef enum
+{
+    SDL_PAN_AXIS_VERTICAL,
+    SDL_PAN_AXIS_HORIZONTAL,
+    SDL_PAN_AXIS_NONE, /**< Indicates this delta describes no axis. Any event marked with this should not rely on delta being sane or meaningful */
```

Comment:
I see only defined and absent form the next PR binding.

Reply:
this can be removed, thanks for catching